src/lib/utils/SynapseClient.ts

```diff
@@ -2563,19 +2582,16 @@ export const searchEntities = (query: SearchQuery, accessToken?: string) => {
   )
 }
 
-/**
- * Get Download List v2
- * http://rest-docs.synapse.org/rest/POST/download/list/query/async/start.html
- */
-export const getAvailableFilesToDownload = (
-  request: AvailableFilesRequest,
-  accessToken: string | undefined = undefined,
-): Promise<DownloadListQueryResponse> => {
-  const downloadListQueryRequest: DownloadListQueryRequest = {
-    concreteType:
-      'org.sagebionetworks.repo.model.download.DownloadListQueryRequest',
-    requestDetails: request,
-  }
+const getDownloadListJobResponse = (
```

Comment:
Reused for all Download List v2 async query job calls.
See http://rest-docs.synapse.org/rest/POST/download/list/query/async/start.html